SimG4CMS/PPS/interface/PPSPixelG4Hit.h

```diff
@@ -0,0 +1,161 @@
+#ifndef _PPS_PixelG4Hit_h
+#define _PPS_PixelG4Hit_h 1
+// -*- C++ -*-
+//
+// Package:     PPS
+// Class  :     PPSPixelG4Hit
+//
+/**\class PPSPixelG4Hit PPSPixelG4Hit.h SimG4CMS/PPS/interface/PPSPixelG4Hit.h
+ 
+ Description: Transient Hit class for PPS taken from those for Calorimeters
+ 
+ Usage: One Hit object should be created
+   -for each new particle entering the calorimeter
+   -for each detector unit (= cristal or fiber or scintillator layer)
+   -for each nanosecond of the shower development
+
+   This implies that all hit objects created for a given shower
+   have the same value for
+   - Entry (= local coordinates of the entrance point of the particle
+              in the unit where the shower starts) 
+   - the TrackID (= Identification number of the incident particle)
+   - the IncidentEnergy (= energy of that particle)
+ 
+*/
+//
+// Original Author:
+//         Created:  Tue May 16 10:14:34 CEST 2006
+//
+
+// system include files
+
+// user include files
+
+#include "G4VHit.hh"
+#include "G4ThreeVector.hh"
+#include <iostream>
+
+class PPSPixelG4Hit : public G4VHit {
+public:
+  // ---------- Constructor and destructor -----------------
+  PPSPixelG4Hit();
+  ~PPSPixelG4Hit() override = default;
+  PPSPixelG4Hit(const PPSPixelG4Hit& right);
+
+  // ---------- operators ----------------------------------
+  const PPSPixelG4Hit& operator=(const PPSPixelG4Hit& right);
+  int operator==(const PPSPixelG4Hit&) { return 0; }
+
+  // ---------- member functions ---------------------------
+  void Draw() override {}
+  void Print() override;
+
+  const G4ThreeVector& getMeanPosition() const { return MeanPosition; };
```

Comment:
Done. Once again, the page says it is "recommended", since it is an old code and due to lack of manpower, this kind of change was put on a lower priority. Hopefully, now all occurrences of recommended change were properly addressed.